src/styles/hartsfield-global.scss

```diff
@@ -0,0 +1,3 @@
+.frosted {
+    background-color: #6d8cb4c3!important;
```

Comment:
@cesarvh - That seems to be an invalid hex code. And, a space is needed after the hex code.